code-reviews/readme.md

```diff
@@ -22,6 +22,22 @@ To ensure that the code review process is healthy and meets the goals stated abo
 2. In the daily standup meeting check tasks pending for review and make sure they have reviewers assigned.
 3. Junior teams and teams new to the process can consider creating separate tasks for reviews together with the tasks themselves.
 
+### Size guidance
+
+We should always aim to have Pull Requests as smaller as possible, without losing context and technical feasibility. Small pull requests have multiple benefits:
+
+1. It's easier to review, a clear benefit for the reviewers.
+2. It's easier to deploy, at least in terms of a bigger Pull Request. This is aligned with the strategy of release fast and release often.
+3. Minimizes possible conflicts, or stale Pull Requests difficult to merge and keep in sync with master either because it's very dynamic or because it has some refactoring.
```

Comment:
```suggestion
3. Minimizes possible conflicts and stale Pull Requests, which are difficult to merge and keep in sync with master either because they're very dynamic or contain refactoring.
```

Reply:
NIT: "Minimize possible conflicts..."